human-kinases/human_kinases.ipynb

```diff
@@ -0,0 +1,3599 @@
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>574</th>\n",
+       "      <td>Q9Y616</td>\n",
+       "      <td>IRAK3</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>575</th>\n",
+       "      <td>Q9Y6E0</td>\n",
+       "      <td>STK24</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>576</th>\n",
+       "      <td>Q9Y6M4</td>\n",
+       "      <td>CSNK1G3</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>577</th>\n",
+       "      <td>Q9Y6R4</td>\n",
+       "      <td>MAP3K4</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>578</th>\n",
+       "      <td>Q9Y6S9</td>\n",
+       "      <td>RPS6KL1</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "      <td>True</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>579 rows × 5 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      UniprotID     Name  kinhub  klifs  pkinfam\n",
+       "0    A0A0B4J2F2    SIK1B   False  False     True\n",
+       "1        A4QPH2  PI4KAP2   False   True    False\n",
+       "2        B5MCJ9   TRIM66    True  False    False\n",
+       "3        O00141     SGK1    True   True     True\n",
+       "4        O00238   BMPR1B    True   True     True\n",
+       "..          ...      ...     ...    ...      ...\n",
+       "574      Q9Y616    IRAK3    True   True     True\n",
+       "575      Q9Y6E0    STK24    True   True     True\n",
+       "576      Q9Y6M4  CSNK1G3    True   True     True\n",
+       "577      Q9Y6R4   MAP3K4    True   True     True\n",
+       "578      Q9Y6S9  RPS6KL1    True   True     True\n",
+       "\n",
+       "[579 rows x 5 columns]"
+      ]
+     },
+     "execution_count": 103,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "grouped = kinases.groupby([\"UniprotID\", \"Name\"]).agg('sum').apply({'kinhub': bool, 'klifs':bool, 'pkinfam':bool})\n",
+    "grouped.reset_index(level=grouped.index.names, inplace=True)\n",
+    "grouped"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Different databases assign different names to each Uniprot ID ¬¬"
```

Comment:
Could we add them all as a list in the table, would be super useful for later mappings